tests/test_visitors/test_ast/test_functions/test_super_call.py

```diff
@@ -56,9 +73,24 @@ def some_method(self, arg1):
         super(Example, self).some_method(arg1)
 """
 
+super_call_with_wrong_method = """
+class Example(Parent):
+    def some_method(self):
+        super().other_method()
+"""
+
+super_call_with_wrong_property = """
+class Example(Parent):
+    @property
+    def x(self):
+        return super().y
```

Comment:
Can we please also test:

- `super().prop.nested`
- `super().prop.method()`
- `super().prop['key']`

We also need to cover all the similar cases for both:
- `super()`
- `super(Class, self)`

I suggest to parametrise your examples like so:

```
super_call_with_method = """
class Example(Parent):
    def some_method(self):
        super({0}).{1}
"""
```

And in tests use two fixtures: `''` and `Class, self` for the first (`{0}`) indexed param. 
And create a second fixture that will insert different properties:

- `.prop`
- `.method()`
- `.prop.nested`
- etc

This way we will cover everything.

Reply:
The same applies for both correct and wrong examples.